cheat.php

```diff
@@ -148,7 +150,17 @@
 		'{normal} - Difficulty: {yellow}' . GetNameForDifficulty( $Zone )
 	);
 
-	$SkippedLagTime = floor( curl_getinfo( $c, CURLINFO_TOTAL_TIME ) - curl_getinfo( $c, CURLINFO_STARTTRANSFER_TIME ) );
+	if ( !isset( $RepositoryScriptHash ) || $LocalScriptHash === $RepositoryScriptHash )
+		$RepositoryScriptHash = GetRepositoryScriptHash( );
+
+	if ( $LocalScriptHash !== $RepositoryScriptHash )
+	{
+		Msg('-- {green}Repository script has been modified. Make sure to check it for updates.');
+	}
+
+	$SkippedLagTime = curl_getinfo( $c, CURLINFO_TOTAL_TIME ) - curl_getinfo( $c, CURLINFO_STARTTRANSFER_TIME );
+	$SkippedLagTime += curl_getinfo( $c_r, CURLINFO_TOTAL_TIME ) - curl_getinfo( $c_r, CURLINFO_STARTTRANSFER_TIME );
```

Comment:
Don't put the github check here, we don't want to modify the actual script logic just to account for the github check.

Move it before `Waiting seconds before rescanning planets...` message

Reply:
We won't be able to use the skip lag time without $c_r being a global. Proceed?

Reply:
We don't need to if you move the check code where I suggested.